__tests__/app.test.js

```diff
@@ -36,4 +36,42 @@ describe("GET /api" ,() => {
     expect(res.body.endpoints).toEqual(JSON.parse(jsonFileData))
     })
     })
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("status: 200, responds with an article of a specific ID from the GET request", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toMatchObject( {
+                article: {
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+                }
+            })
+        })
+    })
+    test("status: 404, responds with 'The request path does not exist' if the endpoint does not exist", () => {
+        return request(app)
+        .get("/api/artivles/1")
+        .expect(404)
+        .then((res) => {
+            expect(res.body.msg).toBe('The request path does not exist')
+        })
+    })
+    test("status: 404, responds with 'Article not found', when the article ID does not exist", () => {
+        return request(app)
+        .get("/api/articles/99999")
+        .expect(404)
+        .then((res) => {
+            expect(res.body.msg).toBe('Article not found')
+        })
+    })
```

Comment:
You need another test here for if the article_id is not valid (eg. a string rather than a number)